test/functional/rpc_psbt.py

```diff
@@ -192,12 +192,20 @@ def run_test(self):
         psbt_orig = self.nodes[0].createpsbt([{"txid":txid1,  "vout":vout1}, {"txid":txid2, "vout":vout2}], {self.nodes[0].getnewaddress():25.999})
 
         # Update psbts, should only have data for one input and not the other
-        psbt1 = self.nodes[1].walletprocesspsbt(psbt_orig)['psbt']
+        psbt1 = self.nodes[1].walletprocesspsbt(psbt_orig, False, "ALL")['psbt']
```

Comment:
why is it important that signing is `False` and specified sighash flag??

Reply:
I vaguely remember running into a problem specific to sign is `False`, but I can't remember.

Reply:
Because otherwise the test runs into #18039